bigtable/google/cloud/bigtable/table.py

```diff
@@ -276,6 +288,32 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         # We expect an iterator of `data_messages_v2_pb2.ReadRowsResponse`
         return PartialRowsData(response_iterator)
 
+    def mutate_rows(self, rows):
+        """Mutates multiple rows in bulk.
+
+        The method tries to update all specified rows.
+        If some of the rows weren't updated, it would not remove mutations.
+        They can be applied to the row separately.
+        If row mutations finished successfully, they would be cleaned up.
+
+        :type rows: list
+        :param rows: List or other iterable of :class:`.DirectRow` instances.
+
+        :rtype: list
+        :returns: A list of corresponding to each row statuses.
+        """
+        mutate_rows_request = _mutate_rows_request(self.name, rows)
+        client = self._instance._client
+        responses = client._data_stub.MutateRows(mutate_rows_request)
+
+        responses_statuses = [None for _ in range(len(rows))]
```

Comment:
- If `rows` is an iterator, you may not be able to call `len` (you could address this by getting `num_rows` as a return value from `_mutate_rows_request` or using `len(mutate_rows_request.entries)`)
- If `rows` is very large you'd much rather use `six.moves.xrange` than `range`